src/main/java/org/example/examplefinalProject/entity/Parent.java

```diff
@@ -20,14 +21,13 @@ public class Parent {
     @ManyToMany(fetch = FetchType.EAGER, cascade = {CascadeType.DETACH, CascadeType.MERGE, CascadeType.PERSIST, CascadeType.REFRESH})
     @JoinTable(
             name = "student_parent",
-            joinColumns = @JoinColumn(name = "parentId"),
-            inverseJoinColumns = @JoinColumn(name = "studentId")
+            joinColumns = @JoinColumn(name = "parent_id"),
+            inverseJoinColumns = @JoinColumn(name = "student_id")
     )
     @OrderBy(value = "studentSurname, studentName")
+    @JsonManagedReference
     private List<Student> children;
 
-    // Getters and setters...
```

Comment:
pomocný text si vymazal? mohu vědět proč?

Reply:
myslel som ze to je text od chatgpt ak to tam chcete tak ponecham.